pkgs/build-support/go/gomod2nix/symlink/symlink.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
```

Comment:
```suggestion
```
ioutil is deprecated https://pkg.go.dev/io/ioutil and using it in new and shiny features is something we probably should not do.